CHANGELOG.md

```diff
@@ -8,11 +8,13 @@
 - **Change:** Added the default implementation for external link handler. Using [custom tab](https://developers.google.com/web/android/custom-tabs).
 - **Change:** `isTestMode` has been deprecated and replaced with `isPreviewMode` to adopt the Preview mode. See [this](miniapp/USERGUIDE.md#2-configure-sdk-settings-in-androidmanifestxml).
 - **Change:** `com.rakuten.tech.mobile.ras.AppId` has been deprecated, use `com.rakuten.tech.mobile.ras.ProjectId` instead. See [this](miniapp/USERGUIDE.md#2-configure-sdk-settings-in-androidmanifestxml).
+- **Feature:** Added `rakuten.miniapp.device.location` custom permission.
```

Comment:
It is `rakuten.miniapp.device.LOCATION` right ?

Reply:
Ah yes, I haven't noticed it.